gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/grpc.py.j2

```diff
@@ -55,8 +55,8 @@ _LOGGER = std_logging.getLogger(__name__)
 
 class _LoggingClientInterceptor(grpc.UnaryUnaryClientInterceptor):  # pragma: NO COVER
     def intercept_unary_unary(self, continuation, client_call_details, request):
-        request_metadata = client_call_details.metadata
-        if CLIENT_LOGGING_SUPPORTED and _LOGGER.isEnabledFor(std_logging.DEBUG):
+        if CLIENT_LOGGING_SUPPORTED and not _LOGGER.isEnabledFor(std_logging.NOTSET):  # pragma: NO COVER
```

Comment:
Since we're doing this the same check twice in this function, we should do it once and save the result in a variable, and use the variable in the two `if` statements.

[That said, we're only adding the interceptor if the check passes (line 261), so having the check both there and here is redundant. But I think it's worth keeping this check.]

Reply:
addressed.